Fibonachi.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class Fibonachi {
+  public static void main(String[] args) {
+    System.out.print("Enter the number: ");
+    Scanner in = new Scanner(System.in); 
+    int input_number = in.nextInt();
```

Comment:
based on the task specification you must check that input value is none-negative